### DISCUSSION TITLE
OMNeT++ IDE doesn't open files in editor [Windows]

### DISCUSSION THREAD

**Discussion by gabrielbergoc** [omnetpp/omnetpp#1081] [Q&A]
So, I just installed OMNeT++ in Windows 10 (according to INSTALL instructions), and started following the tutorial, but the IDE isn't opening files in its editor... When I double click a file or click with right button and click "Open", the Project Explorer is hid and nothing happens... Not even an error message, either in a dialog or in the shell that is running omnetpp...
Only once, when I chose "Open With > In-Place Editor", an error dialog opened, but I can't seem to reproduce this behavior.

I appreciate any help, thanks in advance

**Comment by rhornig** [comment]
This looks like an Issue, so moving it to the issue tracker (see #1082). Could you please go the workspace `.metadata` folder, delete the `.log` file and then try again to open the IDE, and then the file. That will write the error into the `.log` file, then attach the file here.
